packages/ibm-products/package.json

```diff
@@ -78,32 +78,32 @@
     "classnames": "^2.5.1",
     "copyfiles": "^2.4.1",
     "cross-env": "^7.0.3",
-    "fs-extra": "^11.2.0",
+    "fs-extra": "^11.3.0",
     "glob": "^11.0.1",
     "jest": "^29.7.0",
     "jest-config-ibm-cloud-cognitive": "^1.19.0-rc.0",
     "jest-environment-jsdom": "^29.7.0",
     "namor": "^1.1.2",
-    "npm-check-updates": "^17.1.13",
+    "npm-check-updates": "^17.1.14",
     "npm-run-all": "^4.1.5",
     "rimraf": "^6.0.1",
-    "rollup": "^4.30.1",
+    "rollup": "^4.34.7",
     "rollup-plugin-strip-banner": "^3.1.0",
-    "sass": "^1.83.4",
+    "sass": "^1.85.0",
     "typescript-config-carbon": "^0.3.0",
     "yargs": "^17.7.2"
   },
   "dependencies": {
-    "@babel/runtime": "^7.26.0",
+    "@babel/runtime": "^7.26.9",
     "@carbon/feature-flags": "^0.24.0",
     "@carbon/ibm-products-styles": "^2.56.0-rc.0",
     "@carbon/telemetry": "^0.1.0",
     "@dnd-kit/core": "^6.3.1",
     "@dnd-kit/modifiers": "^9.0.0",
     "@dnd-kit/sortable": "^10.0.0",
     "@dnd-kit/utilities": "^3.2.2",
-    "@ibm/telemetry-js": "^1.8.0",
-    "framer-motion": "^6.5.1 < 7",
+    "@ibm/telemetry-js": "^1.9.1",
+    "framer-motion": "^12",
```

Comment:
We'll just need to manually downgrade this again

Reply:
blah good call! i need to remember that one.

Reply:
@davidmenendez let’s just add it to the upgrade exclusions[^1] until we fully remove `framer-motion`.

[^1]: https://github.com/carbon-design-system/ibm-products/pull/6913/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65